src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -30,9 +31,159 @@ namespace {
 
 namespace query = xrt_core::query;
 using key_type = query::key_type;
+xclDeviceHandle handle;
+xclDeviceInfo2 deviceInfo;
 
 static std::map<query::key_type, std::unique_ptr<query::request>> query_tbl;
 
+static zynq_device*
+get_edgedev(const xrt_core::device* device)
+{
+  return zynq_device::get_dev();
+}
+
+struct devInfo
+{
+   static boost::any
+    get(const xrt_core::device* device,key_type key)
+  {
+    auto edev = get_edgedev(device);
+    switch (key)
+    {
+    case key_type::edge_vendor:
+      return deviceInfo.mVendorId;
+    case key_type::rom_vbnv:
+      return std::string(deviceInfo.mName);
+    case key_type::rom_ddr_bank_size:
```

Comment:
For this key, it might be best to rename it to represent what the "size" represents.  Is it Kbytes, bytes, MBytes, etc.

Reply:
Here I am using the key which is already defined in query_requests.h and as the to_string method in that key structure uses  "value << 30" we can infer that it is expecting a Gbytes value which  it is trying to convert into bytes by using left shift.
I dint change the name as it is already in use by pcie platform.

Reply:
O.K. Then we need to update the PCI platforms as well.